message.module

```diff
@@ -15,10 +16,10 @@ use Drupal\message\Entity\MessageType;
  *
  * Handles messages deletion when referenced entities are being deleted.
  *
- * @param \Drupal\Core\Entity\EntityInterface $entity
```

Comment:
I know it works, question is what is the standard -- you look for examples
in core
On Apr 4, 2016 8:17 AM, "HelenaEksler" <notifications@github.com> wrote:

> In message.module
> <https://github.com/Gizra/message/pull/97#discussion_r58329066>:
>
> > @@ -15,10 +16,10 @@ use Drupal\message\Entity\MessageType;
> >   *
> >   * Handles messages deletion when referenced entities are being deleted.
> >   *
> > - * @param \Drupal\Core\Entity\EntityInterface $entity
>
> I'm pretty sure there is no need since we've added it to use -
> https://github.com/Gizra/message/pull/97/files#diff-e1850e575cf08eedf0ba88d0179db1b2R8
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/Gizra/message/pull/97/files/29c79202f3d5bc1dcb28ab1d2bf405f87a7c0e97#r58329066>
>